api/app/src/command/medical/organization/update-organization.ts

```diff
@@ -1,23 +1,24 @@
 import { Organization, OrganizationData } from "../../../models/medical/organization";
-import NotFoundError from "../../../errors/not-found";
+import { validateVersionForUpdate } from "../../../models/_default";
+import { getOrganizationOrFail } from "./get-organization";
 
-export const updateOrganization = async ({
-  id,
-  cxId,
-  data,
-}: {
-  id: string;
-  cxId: string;
-  data: OrganizationData;
-}): Promise<Organization> => {
-  const [count, rows] = await Organization.update(
-    {
-      data,
+export type OrganizationUpdateCmd = Pick<Organization, "id" | "cxId"> &
+  Partial<Pick<Organization, "version">> &
+  OrganizationData;
+
+export const updateOrganization = async (
+  orgUpdate: OrganizationUpdateCmd
+): Promise<Organization> => {
+  const { id, cxId, version, name, type, location } = orgUpdate;
+
+  const org = await getOrganizationOrFail({ id, cxId });
+  validateVersionForUpdate(org, version);
```

Comment:
Need to load and call a validation function explicitly, I couldn't make sequelize to validate the version.